Binary files /dev/null and b/node_modules/react-native-reanimated/android/build/tmp/compileReleaseJavaWithJavac/previous-compilation-data.bin differ
diff --git a/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedModule.java b/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedModule.java
index 35b2f96..8a3b714 100644
--- a/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedModule.java
+++ b/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedModule.java
@@ -115,12 +115,12 @@ public class ReanimatedModule extends ReactContextBaseJavaModule
     // Keep: Required for RN built in Event Emitter Calls.
   }
 
-  @Override
-  public void onCatalystInstanceDestroy() {
-    super.onCatalystInstanceDestroy();
-
-    if (mNodesManager != null) {
-      mNodesManager.onCatalystInstanceDestroy();
-    }
-  }
+  // @Override
+  // public void onCatalystInstanceDestroy() {
+  //   super.onCatalystInstanceDestroy();
+
+  //   if (mNodesManager != null) {
+  //     mNodesManager.onCatalystInstanceDestroy();
+  //   }
+  // }
 }
diff --git a/node_modules/react-native-reanimated/android/src/reactNativeVersionPatch/ReanimatedUIManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedUIManager.java b/node_modules/react-native-reanimated/android/src/reactNativeVersionPatch/ReanimatedUIManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedUIManager.java
index 2c6cb18..f42b6b6 100644
--- a/node_modules/react-native-reanimated/android/src/reactNativeVersionPatch/ReanimatedUIManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedUIManager.java
+++ b/node_modules/react-native-reanimated/android/src/reactNativeVersionPatch/ReanimatedUIManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedUIManager.java
@@ -100,7 +100,7 @@ public class ReanimatedUIManager extends UIManagerModule {
   @ReactMethod
   @Deprecated
   public void replaceExistingNonRootView(int oldTag, int newTag) {
-    super.replaceExistingNonRootView(oldTag, newTag);
+    // super.replaceExistingNonRootView(oldTag, newTag);
   }
 
   /**
@@ -113,7 +113,7 @@ public class ReanimatedUIManager extends UIManagerModule {
   @ReactMethod
   @Deprecated
   public void removeSubviewsFromContainerWithID(int containerTag) {
-    super.removeSubviewsFromContainerWithID(containerTag);
+    // super.removeSubviewsFromContainerWithID(containerTag);
   }
 
   /**
